### ISSUE TITLE
New Task definition policy: enforce trusted artifacts convention

### ISSUE THREAD

**Issue by zregvart** [enterprise-contract/ec-policies#876]
We should enforce the trusted artifacts result/parameter naming convention. That is:

Any step utilizing the `quay.io/redhat-appstudio/build-trusted-artifacts` image should have it's positional arguments in the form of `$([params|results].*_ARTIFACT)=<any value>`.

This rule should be present in the `redhat` collection.

**Comment by lcarva** [comment]
@zregvart, this would be policy rules on the task namespace, correct? Since the image value cannot be obtained reliably from a pod executed by cri-o, we cannot rely on this information from the slsa provenance (release namespace policies).

**Comment by zregvart** [comment]
Yes, this is about validating the Task definitions not attestations.

**Comment by lcarva** [comment]
https://github.com/enterprise-contract/ec-policies/pull/1014

**Comment by robnester-rh** [comment]
Resolved in #1014 and konflux/build-definitions#1049
